src/components/TableCard/TableCard.jsx

```diff
@@ -623,6 +642,7 @@ const TableCard = ({
           <StyledStatefulTable
             columns={columnsToRender}
             data={tableDataWithTimestamp}
+            id={`table-for-card-${id}`}
```

Comment:
I assume this is for testing - could we use `data-testid` instead?

Reply:
we could but this is primary just to fix the duplicate ids issue in the DOM when multiple tablecards were in the same view.  since they can configure the id of the card they'll still be able to locate the correct DOM elements